docs/en/advanced_guides/datasets/nuscenes_det.md

```diff
@@ -61,69 +61,50 @@ Here, .pkl files are generally used for methods involving point clouds and coco-
 Next, we will elaborate on the details recorded in these info files.
```

Comment:
We need to remove all the json files in the directory

Reply:
remove ` Here, .pkl files are generally used for methods involving point clouds and coco-style .json files are more suitable for image-based methods, such as image-based 2D and 3D detection.`